lib/src/logic.dart

```diff
@@ -678,7 +678,8 @@ class Logic {
   /// nextVal <= val.getRange(0, 6); // = val.slice(0, -2) & output: 0b001110, where the output.width=6
   /// ```
```

Comment:
Make sure to update the doc comments about behavior if `endIndex` is not provided